java-test-samples/java-appsync-sam/README-Cognito.md

```diff
@@ -0,0 +1,49 @@
+# Shared Resources
```

Comment:
I recommend adding a paragraph to describe the purpose of the app and what it does. Would help to provide more context. Perhaps start with an **Overview** section?